res/unix/Makefile

```diff
@@ -1,22 +1,42 @@
-POLARIS_BIN_DIR := ~/.local/bin/polaris
-POLARIS_DATA_DIR := ~/.local/share/polaris
+.PHONY: all build install install-bin install-data clean uninstall
+
+BINDIR ?= /usr/local/bin
+DATADIR ?= /usr/local/share
```

Comment:
```suggestion
PREFIX ?= /usr/local
BINDIR ?= $(PREFIX)/bin
DATADIR ?= $(PREFIX)/share
```

Reply:
I just pushed some follow-up changes. Your suggestion lead me to following the [Make manual](https://www.gnu.org/software/make/manual/html_node/Directory-Variables.html#Directory-Variables) more closely. The data/cache/log directories are now also under `/usr/local` by default, but can be overridden by setting `LOCALSTATEDIR`.

Tests should work regardless of all this as they do not rely on any of these values.